src/components/EditCoursePage/CollapsibleCourseRun.jsx

```diff
@@ -450,6 +456,7 @@ CollapsibleCourseRun.defaultProps = {
   courseSubmitInfo: {
     isSubmittingRunReview: false,
   },
+  editable: true,
```

Comment:
Just want to call out and make sure having it default to `true` is intentional. If for whatever reason, our data is messed up, I think I'm of the opinion we should lean towards locking the form.